### DISCUSSION TITLE
IBC

### DISCUSSION THREAD

**Discussion by qdrn** [massalabs/massa#3508] [Ideas]
It would be nice to study the possibility to integrate IBC to Massa. This would (to my understanding) enable trustless bridges between blockchains from the Cosmos ecosystem and Massa.

From initial discussions it appears that it would be difficult for Massa due to the peculiarities of the consensus. Maybe you can summarise the issues here @damip ? Feel free to edit my message to keep the discussion clean.

Relevant links:
- https://github.com/informalsystems/hermes
- https://forum.tezosagora.org/t/tenderbake-connection-to-ibc-cosmos-ecosystem-possible/3809/4

**Comment by RuslanGlaznyov** [comment]
Is there any work around this?
